docs/gettingstarted/query_test_report.rst

```diff
@@ -63,6 +63,17 @@ To print just the raw format fields use the command ``buildtest report --formatf
 
     .. command-output:: buildtest report --formatfields
 
+Format Reports (``buildtest report --detailed``)
+-----------------------------------------------
+
+The **buildtest report --helpformat** command displays a list of all available format fields, which can be quite long.
+If you want to format the report table by detailed fields, run the below command. These detailed fields are user, state, id, name, buildspecs, errfile, outfile, runtime, and returncode.
+You can run buildtest report --detailed command as shown below.
+
```

Comment:
What do you think? 
<img width="1173" alt="Screen Shot 2023-07-11 at 1 29 04 PM" src="https://github.com/buildtesters/buildtest/assets/76664109/4b8ce796-cda4-4ae0-9a8e-b2a0b2d67e80">

Reply:
This statement `buildtest report -d is an alias for ...` is not correct, please remove this. It just a short option. We dont need to document this since we have many short options.

For the text you can say the following




The `buildtest report --detailed` option can be used to show a detailed test report that may be of interest when examining a test. This option is synonymous to running  `buildtest report --format name, id, user, state, returncode, runtime, outfile, errfile, buildspec`. Shown below is an example output.

Reply:
Fixed, thank you!